docs/integrations/sources/surveymonkey.md

```diff
@@ -58,35 +74,36 @@ Please read this [docs](https://developer.surveymonkey.com/api/v3/#getting-start
 
 The SurveyMonkey API applies heavy API quotas for default private apps, which have the following limits:
 
-* 125 requests per minute
-* 500 requests per day
+* 125 requests per minute.
```

Comment:
I think it's fine to leave "per" here, but I just wanted to point out that "per" is a Latinism. Non-English expressions can be problematic if we ever want to localize our content in the future.